.changes/v1.0.0/23-features.md

```diff
@@ -0,0 +1,2 @@
+- **New Resource:** `vcfa_certificate_library` to manage Certificate Libraries [GH-23]
+- **New Data Source:** `vcfa_certificate_library` to read Certificate Libraries [GH-23]
```

Comment:
"to manage/read Certificate Libraries" reads strange. What about "to manage/read certificates in a certificate library"?

Reply:
Changed to `vcfa_certificate` and applied this suggestion